frontend/src/components/videoPlayer/AddComment.js

```diff
@@ -20,27 +24,99 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const AddComment = () => {
+const AddComment = ({ data }) => {
   const classes = useStyles();
+  const [comment, setComment] = useState('');
+  const [alert, setAlert] = useState({
+    show: false,
+    message: '',
+    severity: '',
+  });
+
+  const onChangeComment = (e) => {
+    const comment = e.target.value;
+    setComment(comment);
+  };
+
+  const handleComment = async (e) => {
+    e.preventDefault();
+    try {
+      await movieService.createComment(comment, data.movie.imdbCode);
+      setAlert({
+        show: true,
+        message: 'Thanks for the comment!',
+        severity: 'success',
```

Comment:
remember the users have profiles in different languages, error message needs to be translated to 4 languages